### PR TITLE
Provider: Use context-aware copy when no results are present

### PR DESCRIPTION
**Pull request by tyom**
Cases view.

PT:86781444

### CODE REVIEW

**Comment by teneightfive** [comment]
ok to test

File: tests/angular-js/e2e/providerCase.js [hunk 1]
```diff
@@ -78,15 +78,7 @@
...
 
-        // click but cancel accept
-        close_button.click();
-        browser.switchTo().alert().dismiss();
-
-        expect(close_button.isPresent()).toBe(true);
-
-        // click and accept case
         close_button.click();
-        browser.switchTo().alert().accept();
```
Comment (teneightfive) [inline tests/angular-js/e2e/providerCase.js]: Shouldn't these changes be on the other branch?

Comment (tyom) [inline tests/angular-js/e2e/providerCase.js]: Oh yes! Well spotted.

**Comment by tyom** [comment]
retest this please

**Comment by teneightfive** [comment]
Just had a thought about this. It still says the search criteria copy when a provider has selected a tab and gets no results. Technically it is actually a search as we're adding search options but it was here where people were getting fused and thinking they had searched a name in the top search bar.

Maybe we find some better copy to always appear here that doesn't use the term 'search criteria'?

**Comment by tyom** [comment]
It shouldn't be saying “search criteria” when you select one of the tabs. As per logic in templates it only shows the word “search” when `searchParams.search` is not `null`.

Do you have a different copy in mind?

**Pull request merged**